scripts/coverage.sh

```diff
@@ -79,7 +79,7 @@ NPROC=$(nproc)
 JOBS=$((JOBS>NPROC ? NPROC : JOBS))
 
 RUST_LOG=solana=trace _ "$cargo" nightly test --jobs "$JOBS" --target-dir target/cov --no-run "${packages[@]}"
-if RUST_LOG=solana=trace _ "$cargo" nightly test --jobs "$JOBS" --target-dir target/cov "${packages[@]}" -- --nocapture 2> >(tee target/cov/coverage-stderr.log >&2); then
+if RUST_LOG=solana=trace _ ci/upload-console.sh "$cargo" nightly test --jobs "$JOBS" --target-dir target/cov "${packages[@]}" -- --nocapture --color=always; then
```

Comment:
todo: remove `--color=always`; this isn't working.

Reply:
better for the raw log view anyway!